src/jvmTest/resources/test.sarif.json

```diff
@@ -21,6 +21,14 @@
                   "startColumn": 1,
                   "startLine": 1
                 }
+              },
+              "properties": {
```

Comment:
Add explicit deserialisation tests for
*  empty map `{}`,
* tag only `{"tags": [...]}`,
* and `{"foo": null}`?

Reply:
I'll  create unit tests just to ensure how `PropertyBag` works.

Reply:
yep, that's what I meant, it's a start on increasing the coverage! And this is actually a complicated part.

Reply:
Done!